src/main/java/httpserver/utils/constants.java

```diff
@@ -0,0 +1,5 @@
+package httpserver.utils;
+
+public class constants {
+    public static String LINE_BREAK = "\r\n\r\n";
```

Comment:
Ohh I like this. Keeping magic numbers in a specific spot, even if the name of the class could change later once you see what else goes here.